frontend/src/components/account/Orders/components/OrdersMobile/OrdersMobile.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ICONS } from '../../../../ui-kit/icons';
+import styles from './OrdersMobile.module.scss';
+import { useNavigate } from 'react-router-dom';
+
+const OrdersMobile = () => {
+  const navigate = useNavigate();
```

Comment:
Delete this component. You have this info in TitleMobile

Reply:
Done